### PR TITLE
docs: add Attention message for 'response-rewrite' doc.

### PR DESCRIPTION
**Pull request by Miss-you**
### What this PR does / why we need it:
1. add Summary for 'zh-cn/plugins/response-rewrite.md'
2. add Attention message for 'response-rewrite' doc.

ref:
https://github.com/apache/apisix/issues/2721

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

File: doc/plugins/response-rewrite.md [hunk 1]
```diff
@@ -116,3 +117,11 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 ```
 
 The `response rewrite` plugin has been disabled now. It works for other plugins.
+
+## Attention
+
+`ngx.exit` will interrupt the execution of the current request and return status code to Nginx.
+
+![](https://cdn.jsdelivr.net/gh/Miss-you/img/picgo/20201113010623.png)
```
Comment (juzhiyuan) [inline doc/plugins/response-rewrite.md:125]: Not sure if this image link is stable, could we put it in the images folder?

File: doc/plugins/response-rewrite.md [hunk 2]
```diff
@@ -116,3 +117,11 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
...
+
+However, if you execute `ngx.exit` during the access phase, it only interrupts the request processing phase, and the response phase will still process it, i.e. if you configure the `response-rerite` plugin, it will force overwriting of your response information (e.g. response code).
```
Comment (juzhiyuan) [inline doc/plugins/response-rewrite.md:127]: Please check the plugin name

Comment (juzhiyuan) [reply doc/plugins/response-rewrite.md:127]: Not sure if response code is the same as response status code.

File: doc/zh-cn/plugins/response-rewrite.md [hunk 3]
```diff
@@ -19,12 +19,21 @@
...
+- [**禁用插件**](#禁用插件)
+- [**注意事项**](#注意事项)
+
+## response-rewrite
 
 该插件支持修改上游服务返回的 body 和 header 信息。
 
 使用场景：
-1、可以设置 `Access-Control-Allow-*` 等 header 信息，来实现 CORS (跨域资源共享)的功能。
+1、可以设置 `Access-Control-Allow-*` 等 header 信息，来实现 CORS （跨域资源共享）的功能。
```
Comment (juzhiyuan) [inline doc/zh-cn/plugins/response-rewrite.md:36]: Two spaces?

**Review by juzhiyuan — CHANGES_REQUESTED** [review]

**Comment by spacewander** [comment]
Also remove the `upstream` from the doc? Since we don't rewrite upstream response only.

**Comment by liuxiran** [comment]
> Also remove the `upstream` from the doc? Since we don't rewrite upstream response only.

agree with remove `upstream`,  add it back when we think about support it later

File: doc/plugins/response-rewrite.md (hunk 2 above)
Comment (Miss-you) [reply doc/plugins/response-rewrite.md:127]: done


> Please check the plugin name

Comment (Miss-you) [reply doc/plugins/response-rewrite.md:127]: > response status code
response code ->
response status code

**Comment by Miss-you** [comment]
> upstream

`response rewrite plugin, rewrite the content from upstream.`
->
`response rewrite plugin, rewrite the content returned by the upstream as well as Apache APISIX itself.`

File: doc/zh-cn/plugins/response-rewrite.md (hunk 3 above)
Comment (Miss-you) [reply doc/zh-cn/plugins/response-rewrite.md:36]: one space
```
CORS(space)（xxx）

ps: '（） ' is different from '()'
```

**Review by spacewander — APPROVED** [review]

File: doc/zh-cn/plugins/response-rewrite.md [hunk 4]
```diff
@@ -85,3 +94,29 @@ X-Server-status: on
...
+        }
+    }
+}'
+```
+
+## 注意事项
+
+`ngx.exit`将中断当前请求的执行，并返回状态码给 Nginx。
+
+![](https://cdn.jsdelivr.net/gh/Miss-you/img/picgo/20201113010623.png)
```
Comment (juzhiyuan) [inline doc/zh-cn/plugins/response-rewrite.md:120]: Will this link be stable?

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**